Index: src/main/java/action/post/PostFindAction.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package action.post;\n\nimport action.Action;\nimport dao.PostRepository;\nimport dao.PostRepositoryImpl;\n\nimport javax.servlet.http.HttpServletRequest;\nimport javax.servlet.http.HttpServletResponse;\n\npublic class PostFindAction implements Action {\n    @Override\n    public void execute(HttpServletRequest request, HttpServletResponse response) {\n        Integer id=null;\n        try{\n            id=Integer.parseInt(request.getParameter(\"id\"));\n        }catch(NumberFormatException e){\n            id=null;\n        }\n\n        PostRepository postRepository= PostRepositoryImpl.getInstance();\n        request.setAttribute(\"post\",postRepository.findOne(id));\n        request.setAttribute(\"postId\",id);\n        //--test\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/action/post/PostFindAction.java b/src/main/java/action/post/PostFindAction.java
--- a/src/main/java/action/post/PostFindAction.java	(revision 6b88b87b41cc8c6fca8742d6ac6ae60f1d9f3b8a)
+++ b/src/main/java/action/post/PostFindAction.java	(date 1620220273280)
@@ -20,6 +20,5 @@
         PostRepository postRepository= PostRepositoryImpl.getInstance();
         request.setAttribute("post",postRepository.findOne(id));
         request.setAttribute("postId",id);
-        //--test
     }
 }
